packages/babel-preset-env/test/fixtures/plugins-integration/issue-7527/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": [],
+  "presets": ["../../../../lib"]
```

Comment:
shouldnt it be possible to specify preset name here instead of the hardcoded path?

Reply:
I don't know, in the other tests of this package it is done like this.
We should modify our tests to accept _real_ plugin and preset names anyway (with `@babel/plugin-*`).